Setting/Factions/The Coven.md

```diff
@@ -0,0 +1,32 @@
+---
+tier: "5"
+allies:
+  - "[[Scrappers]]"
+enemies:
+  - "[[Ship Defense Corps]]"
+  - Intership Council
+demonym: cadmus
+aliases:
+  - CADMUS
```

Comment:
seems like this was copied from CADMUS. I feel like this makes more sense as like tier 3 at most